deployment/single-server/vars/nlx.example.yml

```diff
@@ -0,0 +1,83 @@
+---
+
+# Example nlx configuration variables
+# See https://github.com/open-zaak/ansible-collection/tree/master/roles/nlx_docker/defaults/main.yml
+# for all available variables
+
+# INWAY
+nlx_inway_version: latest  # see https://hub.docker.com/r/nlxio/inway/tags
+nlx_inway_self_address: inway.gemeente.nl:8443  # domain must be publicly available
+nlx_inway_directory_registration_address: directory-registration-api.demo.nlx.io:443
+
+nlx_inway_services:
+  - name: openzaak-zaken
+    endpoint_url: "https://open-zaak.gemeente.lan/zaken/api/v1"
```

Comment:
Can't we make this a variable as done in the k8s file?

Reply:
Done